templates/form/unzer_pay_later_direct_debit.liquid

```diff
@@ -28,10 +28,13 @@
             <input type="text" id="last_name" class="form-control" name="person[last_name]" value="{{ model.last_name }}" {% if model.last_name %}readonly="true" {% endif %} autocomplete="on" required>
           </div>
         </div>
-        <div class="form-group">
+        <div class="form-group{% if model.invalid_dob? %} has-error{% endif %}">
           <div class="col-xs-12">
             <label for="person_dob" class="control-label">{% t Date of Birth %}</label>
             <input type="date" id="person_dob" name="person[dob]" class="form-control" required>
+            {% if model.invalid_dob? %}
+              <span class="help-block">{% t Invalid Date of Birth. You need to 18 or older %}</span>
```

Comment:
How do you 18?

Reply:
By trying very very hard :)
(it's a misspelling, nice catch)